pages/documentation/settings.md

```diff
@@ -19,17 +21,37 @@ subnav:
   href: '#utilities-settings'
 ---
 
-A good way to get started with USWDS settings is to copy the `dist/theme` directory out of the `uswds` package and into the same directory where you keep your project Sass files. Then use the `@forward` directive in Sass to include these settings **before including the main USWDS Sass**. You can use the `styles.scss` example file in the `dist/theme` directory as a starting point for your own Sass entry point:
+## Introducing USWDS settings
+USWDS is built using a suite of customizable settings that allows you to configure the design system to fit your project's needs. These settings are Sass variables that begin with the `$theme-` prefix and are typically defined with [USWDS design tokens]({{ site.baseurl }}/design-tokens). The full list of USWDS settings and their default values can be found in the [tables below](#general-settings).
+
+## Configuring Custom USWDS settings
+All USWDS settings are defined in the `uswds-core` package and can be configured using Sass' [`@use..with()`](https://sass-lang.com/documentation/at-rules/use#configuration) rule. To create a custom configuration of USWDS, add USWDS settings variables with custom values inside the parentheses of this statement, as shown in this example:
 
 ```scss
-@forward 'uswds-theme';
-@forward 'uswds';
-@forward 'uswds-theme-custom-styles';
+@use "uswds-core" with (
+  $theme-show-compile-warnings: false,
+  $theme-show-notifications: false,
+  $theme-image-path: "../uswds/img",
+  $theme-show-compile-warnings: true,
+  $theme-banner-background-color: "ink",
+  $theme-banner-link-color: "primary-light",
+  $theme-input-tile-border-color: "base",
+);
 ```
+Note that the settings variables in this module inform both general and component customizations.
+
+### What to include in your configuration
+Include only USWDS settings that you wish to modify in this configuration. These items should be comma-separated and given values of the appropriate type.
+
+Please note that this configuration accepts only current USWDS settings variables &mdash; adding anything else here will result in an error.
```

Comment:
Maybe add site-note here to make it standout?

Reply:
There is a more specific site-note about this at the [bottom of this file](https://github.com/uswds/uswds-site/pull/1755/files#diff-d26ac88e392b8b5ae6f8fd8fc17fa7ace2877ed30ea067822dab4e3190a19b32R86-R87) - should it be moved up? Is it redundant? I lean towards putting specific error notes at the bottom of the copy and more casual/generic guidance up top, but I am very open to input here.

Reply:
Update: I kept the current format of having the error site-note at the bottom of the text because I wanted this copy to remain very straightforward. Let me know if you'd like to see something different.

Reply:
No, it makes sense. Thanks for taking another look.